test/findOr.js

```diff
@@ -0,0 +1,66 @@
+import { assert } from 'chai';
+
+import * as RA from '../src';
+
+describe('findOr', function () {
+  context('given a list that has the value to find', function () {
+    context('and that value is truthy', function () {
+      specify('should return the found value', function () {
+        const defaultValue = 'default';
+        const list = [1, 2, 3];
+        const predicate = (val) => val === 1;
+
+        assert.deepEqual(RA.findOr(defaultValue, predicate, list), 1);
+      });
+    });
+
+    context('and that value is NaN', function () {
+      specify('should return the default value', function () {
+        const defaultValue = 'default';
+        const list = [1, 2, 3, NaN];
+        const predicate = (val) => RA.isNaN(val);
+
+        assert.deepEqual(
+          RA.findOr(defaultValue, predicate, list),
+          defaultValue
+        );
+      });
+    });
+
+    context('and that value is null', function () {
+      specify('should return the default value', function () {
+        const defaultValue = 'default';
+        const list = [1, 2, 3, null];
+        const predicate = (val) => val === null;
+
+        assert.deepEqual(
+          RA.findOr(defaultValue, predicate, list),
+          defaultValue
+        );
+      });
+    });
+
+    context('and that value is undefined', function () {
+      specify('should return the default value', function () {
+        const defaultValue = 'default';
+        const list = [1, 2, 3, null];
+        const predicate = (val) => val === undefined;
+
+        assert.deepEqual(
+          RA.findOr(defaultValue, predicate, list),
+          defaultValue
+        );
+      });
+    });
+  });
+
+  context('given a list that does not contain the value to find', function () {
+    specify('should return the default value', function () {
+      const defaultValue = 'default';
+      const list = [1, 2, 3];
+      const predicate = (val) => val === 4;
+
+      assert.deepEqual(RA.findOr(defaultValue, predicate, list), defaultValue);
+    });
+  });
+});
```

Comment:
We're missing a currying test entirely. Checkout [this test](https://github.com/char0n/ramda-adjunct/blob/b95fa04adaed288259947dbac5f0225e0ae138f4/test/pathOrLazy.js#L76) how to create one.